Hardware/Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Hardware",
```

Comment:
Not sure if this matters. Should we define the `platforms`? 

```suggestion
    name: "Hardware",
    platforms: [.iOS(.v13)],
```

Reply:
Good catch. It doesn't hurt to be explicit.